registrations/views.py

```diff
@@ -314,6 +315,7 @@ def get_software_type(channel_id):
             or self.UNCLASSIFIED_MESSAGES_DEFAULT_LABEL,
             "type": 7,  # 7 helpdesk
             "op": str(validated_data.get("helpdesk_operator_id")),
+            "eid": str(registration.id) if registration else None,
```

Comment:
I don't think we should use the registration ID here.

If the idea is to use this field to deduplicate events sent to jembi, multiple incoming messages from this registration will be the same, therefore ignored.

We should rather be using something unique to the incoming message, like a message ID.